tests/test_workspace.py

```diff
@@ -278,6 +279,19 @@ def test_image_from_page_basic(self):
             img, info, exif = ws.image_from_page(pcgts.get_Page(), page_id='PHYS_0017')
             self.assertEquals(info['features'], 'binarized,clipped')
 
+    def test_mets_permissions(self):
+        initLogging()
+        with TemporaryDirectory() as tempdir:
+            ws = self.resolver.workspace_from_nothing(tempdir)
+            ws.save_mets()
+            mets_path = join(ws.directory, 'mets.xml')
+            assert filemode(stat(mets_path).st_mode) == '-rw-rw-r--'
```

Comment:
This test fails for me:

     FAILED tests/test_workspace.py::TestWorkspace::test_mets_permissions - AssertionError: assert '-rw-r--r--' == '-rw-rw-r--'

I am afraid that it won't work without using umask.

Reply:
@stweil Can you test again with f92c5a3c? This incorporates your `umask` approach from #608 with a workaround by @pabs3 from https://github.com/untitaker/python-atomicwrites/issues/42